Buildings/Controls/OBC/CDL/Continuous/MultiSum.mo

```diff
@@ -5,10 +5,11 @@ block MultiSum
   parameter Integer nin(min=0) = 0 "Number of input connections"
     annotation (Dialog(connectorSizing=true), HideResult=true);
   parameter Real k[nin]=fill(1, nin) "Input gains";
-  Interfaces.RealInput u[nin] "Connector of Real input signals"
-    annotation (Placement(transformation(extent={{-140,70},{-100,-70}})));
   Interfaces.RealOutput y "Connector of Real output signal"
     annotation (Placement(transformation(extent={{100,17},{134,-17}})));
+  Interfaces.RealInput u1 "Connector of Real input signal 1"
```

Comment:
@kuzha : This should be u[nin] "Connector of Real input signals" and listed before the output.